### PR TITLE
Move to monorepo

### PR DESCRIPTION
**Pull request by jessestorbeck**
These changes transition "pushkin" from our docs repo into a new monorepo structure holding the entire Pushkin codebase. We've also moved the Pushkin documentation from Gitbook to MkDocs. The new docs are already live at https://pushkin-consortium.github.io/pushkin/. NB: merging this PR will probably(?) take the old docs down.

### CODE REVIEW

**Comment by jessestorbeck** [comment]
@jkhartshorne -- I think [c795aa8](https://github.com/pushkin-consortium/pushkin/pull/253/commits/c795aa8a1ef3a4b7fa18f546902a82022b32064b) should fix it so the gitbook docs stay up after we merge this. I also added a note on the homepage to direct people to the new docs.

**Review by hunterschep — APPROVED** [review]
Approving this after what was discussed at the last pushkin meeting about moving forward with the monorepo & prior testing in developing the monorepo

**Pull request merged**
